src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/plugin/set_preprocess.cpp

```diff
@@ -107,5 +105,3 @@ namespace {
                                 ::testing::ValuesIn(heteroConfigs)),
                         InferRequestPreprocessDynamicallyInSetBlobTest::getTestCaseName);
```

Comment:
`InferRequestPreprocessDynamicallyInSetBlobTest` are based on API 1.0 and should be removed

Reply:
Test is removed

Reply:
I reverted deletion to unblock merge because openvino_contrib and npu tests depend on it